subsys/bluetooth/host/hci_core.c

```diff
@@ -2623,23 +2623,27 @@ static void unpair(u8_t id, const bt_addr_le_t *addr)
 	bt_gatt_clear(id, addr);
 }
 
+#if defined(CONFIG_BT_SMP)
 static void unpair_remote(const struct bt_bond_info *info, void *data)
 {
 	u8_t *id = (u8_t *) data;
 
 	unpair(*id, &info->addr);
 }
+#endif /* defined(CONFIG_BT_SMP) */
```

Comment:
Are these necessary if you use IS_ENABLED further down?

Reply:
> Are these necessary if you use IS_ENABLED further down?

Got